vars/abapEnvironmentPipelineStageIntegrationTests.groovy

```diff
@@ -0,0 +1,24 @@
+import groovy.transform.Field
+import com.sap.piper.Utils
+
+import static com.sap.piper.Prerequisites.checkScript
+
+@Field String STEP_NAME = getClass().getName()
+@Field Set GENERAL_CONFIG_KEYS = []
+@Field STAGE_STEP_KEYS = [
+    'cloudFoundryCreateService'
+]
+@Field Set STEP_CONFIG_KEYS = GENERAL_CONFIG_KEYS.plus(STAGE_STEP_KEYS)
+@Field Set PARAMETER_KEYS = STEP_CONFIG_KEYS
+/**
+ * This stage creates a system for Integration Tests. The (custom) tests themselves can be added via a stage extension.
+ */
+void call(Map parameters = [:]) {
+    def script = checkScript(this, parameters) ?: this
+    def stageName = parameters.stageName?:env.STAGE_NAME
+
+    piperStageWrapper (script: script, stageName: stageName, stashContent: [], stageLocking: false) {
+        cloudFoundryCreateService script: parameters.script
```

Comment:
Will there be a new system created for integration tests?

Reply:
Yes, another system including the addon will be created